### PR TITLE
adding docs for HoldingStandard, freezing of account, and Lockable an…

### PR DESCRIPTION
**Pull request by johan-da**
…d InterfaceKeys modules

### CODE REVIEW

File: docs/2.8.0/docs/daml-finance/concepts/asset-model.rst [hunk 1]
```diff
@@ -101,39 +105,46 @@ The terminology is again borrowed from the real world: our cash or shares are us
...
+
 .. _implementations-1:
 
 Interfaces
 ==========
 
-Holding interfaces are defined in the ``Daml.Finance.Interface.Holding`` package. These include a
-:ref:`base holding interface <module-daml-finance-interface-holding-base-24195>`,
-as well as interface definitions for the above properties.
+Holding interfaces are defined in the ``Daml.Finance.Interface.Holding`` package, and consists of
```
Comment (markus-da) [inline docs/2.8.0/docs/daml-finance/concepts/asset-model.rst:134]:
```suggestion
Holding interfaces are defined in the ``Daml.Finance.Interface.Holding`` package, that consists of
```

**Review by markus-da — APPROVED** [review]
LGTM
Left a minor suggestion.

File: docs/2.8.0/docs/daml-finance/concepts/asset-model.rst [hunk 2]
```diff
@@ -101,39 +105,49 @@ The terminology is again borrowed from the real world: our cash or shares are us
...
 at the same custodian.
 
-These properties are exposed by implementing the corresponding interface
+These properties are exposed by letting a holding template implement the corresponding interfaces
 (:ref:`Fungible <type-daml-finance-interface-holding-fungible-fungible-60176>` and
 :ref:`Transferable <type-daml-finance-interface-holding-transferable-transferable-24986>`,
 respectively).
 
-.. _implementations-1:
+The library distinguishes 4 types of holdings, referred to as :ref:`Holding Standard
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/concepts/asset-model.rst:124]:
```suggestion
The library distinguishes four types of holdings, referred to as :ref:`Holding Standard
```

File: docs/2.8.0/docs/daml-finance/concepts/asset-model.rst [hunk 3]
```diff
@@ -101,39 +105,49 @@ The terminology is again borrowed from the real world: our cash or shares are us
...
+3. `TransferableFungible`: Holdings that are both transferable and fungible.
+4. `BaseHolding`: Holdings that are neither transferable nor fungible.
 
 Interfaces
 ==========
 
-Holding interfaces are defined in the ``Daml.Finance.Interface.Holding`` package. These include a
-:ref:`base holding interface <module-daml-finance-interface-holding-base-24195>`,
-as well as interface definitions for the above properties.
+Holding interfaces are defined in the ``Daml.Finance.Interface.Holding`` package, that consists of
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/concepts/asset-model.rst:135]:
```suggestion
Holding interfaces are defined in the ``Daml.Finance.Interface.Holding`` package. It consists of
```

File: docs/2.8.0/docs/daml-finance/concepts/asset-model.rst [hunk 4]
```diff
@@ -101,39 +105,49 @@ The terminology is again borrowed from the real world: our cash or shares are us
...
+:ref:`transferable <module-daml-finance-interface-holding-transferable-88121>`, and
+:ref:`fungible <module-daml-finance-interface-holding-fungible-63712>`.
+
+.. _implementations-1:
 
 Implementations
 ===============
 
-Implementations are provided in ``Daml.Finance.Holding`` for:
+In ``Daml.Finance.Holding``, we provide implementations for each holding standard:
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/concepts/asset-model.rst:145]:
```suggestion
``Daml.Finance.Holding`` provides implementations for each holding standard:
```

File: docs/2.8.0/docs/daml-finance/concepts/asset-model.rst [hunk 5]
```diff
@@ -204,10 +218,12 @@ A base account implementation is provided in
 
 The account can be created with arbitrary
 :ref:`controllers <type-daml-finance-interface-account-account-controllers-36430>`
-(for incoming and outgoing transfers).
+(for incoming and outgoing transfers). In our examples, we typically let accounts be
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/concepts/asset-model.rst:221]:
```suggestion
(for incoming and outgoing transfers). The examples typically let accounts be
```

File: docs/2.8.0/docs/daml-finance/concepts/asset-model.rst [hunk 6]
```diff
@@ -232,9 +248,9 @@ The Retail Client has an
 the former acting as ``owner`` and the latter as ``custodian``.
 
 Finally, the Retail Client is ``owner`` of a
-:ref:`fungible holding <type-daml-finance-holding-fungible-fungible-28517>` at the Commercial Bank
-(the ``custodian`` in the contract). The holding references the currency instrument, as well as the
-account.
+:ref:`transferable and fungible holding <module-daml-finance-holding-transferablefungible-77726>` at
+the Commercial Bank (i.e., the ``custodian`` in the contract). The holding references the currency
+instrument, as well as the account.
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/concepts/asset-model.rst:253]:
```suggestion
instrument and the account.
```

File: docs/2.8.0/docs/daml-finance/reference/patterns.rst [hunk 7]
```diff
@@ -68,7 +68,7 @@ To do that, we define a `Reference` template that
  - is keyed by the `InstrumentKey`
  - contains the interface contract id of the target instrument
 
-We can then fetch an interface "by key" by 
+We can then fetch an interface "by key" by
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/reference/patterns.rst:71]:
```suggestion
You can then fetch an interface "by key" by
```

File: docs/2.8.0/docs/daml-finance/reference/patterns.rst [hunk 8]
```diff
@@ -77,9 +77,10 @@ We can then fetch an interface "by key" by
 
 Similarly, this pattern also lets us exercise a choice on an interface "by key".
 
-The Reference pattern is currently used in Daml Finance for instruments and accounts, where we
-ensure that a template and its companion `Reference` template are kept in sync. It is however
-important to understand this pattern should you implement custom instruments or accounts.
+The Reference pattern is currently used in Daml Finance for instruments, accounts, and the holding
+factory, where we ensure that a template and its companion `Reference` template are kept in sync.
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/reference/patterns.rst:81]:
```suggestion
factory, where a template and its companion `Reference` template are kept in sync.
```

File: docs/2.8.0/docs/daml-finance/reference/patterns.rst [hunk 9]
```diff
@@ -77,9 +77,10 @@ We can then fetch an interface "by key" by
...
+It is however important to understand this pattern should you implement custom instruments,
```
Comment (loiskilberg-da) [inline docs/2.8.0/docs/daml-finance/reference/patterns.rst:82]:
```suggestion
It is important, however, to understand this pattern should you implement custom instruments,
```

**Review by loiskilberg-da — APPROVED** [review]
Looks good! Left a few minor grammar suggestions.

File: docs/2.8.0/docs/daml-finance/concepts/asset-model.rst (hunk 5 above)
Comment (johan-da) [reply docs/2.8.0/docs/daml-finance/concepts/asset-model.rst:221]: changed it to: "Our examples typically let accounts be .."

File: docs/2.8.0/docs/daml-finance/reference/patterns.rst (hunk 8 above)
Comment (johan-da) [reply docs/2.8.0/docs/daml-finance/reference/patterns.rst:81]: I changed the this paragraph to:

The Reference pattern is currently used in Daml Finance for instruments, accounts, and the holding
factory. To maintain synchronization between a template and its corresponding `Reference`, we 
simultaneously create or archive instances of both. Additionally, if the contract id of the target 
template changes, we update the `Reference` instance accordingly.

It is important to understand this pattern should you implement custom instruments, accounts or 
holding factories.

**Pull request merged**